near-sdk/Cargo.toml

```diff
@@ -23,8 +23,8 @@ near-sdk-macros = { path = "../near-sdk-macros", version = "0.11.0"}
 borsh = "0.6.1"
 bs58 = "0.3"
 base64 = "0.11.0"
-near-vm-logic = "0.9.1"
-near-runtime-fees = "0.9.1"
+near-vm-logic = {path = "../../nearcore/runtime/near-vm-logic", version = "0.9.1"}
```

Comment:
Please change this back to versions when PR is finished (versions of these packages should increase on nearcore side)